src/Vizkit3dWorld.cpp

```diff
@@ -18,8 +18,12 @@
 
 namespace vizkit3d_world {
 
-Vizkit3dWorld::Vizkit3dWorld(std::string path, std::vector<std::string> modelPaths,
-                            int cameraWidth, int cameraHeight, double horizontalFov, double zNear, double zFar)
+Vizkit3dWorld::Vizkit3dWorld(std::string path, 
+                            std::vector<std::string> modelPaths,
+                            std::vector<std::string> exclude_from_sdf,
```

Comment:
One golden rule about naming convention: stick to whatever the package already chose. Here, this means that you should really use lowerCamelCase for parameter names.